felt_python/elements.py

```diff
@@ -1,87 +1,162 @@
 """Elements and element groups"""
 
 import json
+from typing import Dict, Any, List, Union
```

Comment:
I don't think we need to import `typing` here, we can continue using the "simple" syntax with the standard python classes (`dict` instead of `typing.Dict`, `|` instead of `typing.Union`, etc)